push.php

```diff
@@ -116,3 +117,51 @@ function zw_webapp_show_debug_message(WP_Post $post)
         delete_post_meta($post->ID, 'zw_webapp_debug_msg');
     }
 }
+
+function zw_webapp_get_daily_push_count()
+{
+    $cache_key = 'zw_webapp_daily_push_count';
+    $daily_push_count = wp_cache_get($cache_key);
+
+    if (false === $daily_push_count) {
+        $daily_push_count = array();
+
+        for ($i = 0; $i <= 6; $i++) {
+            $date_query = array(
```

Comment:
Would prefer seeing `$date = strtotime('-' . $i . ' days');`, and have the `date()` invocations below all use the same date.